src/views/Public/Index.vue

```diff
@@ -3,15 +3,79 @@
     <public-header :page="page"
                    :namespace="namespace"
                    @toggleNav="navVisible = $event" />
-    <router-view :namespace="namespace"
+    <div v-if="showSteps" class="d-flex flex-column m-5 vh-75">
```

Comment:
Why did you not make a component for this?

Reply:
That is the plan. After i get the goahead for this style of onboarding. 
Didnt want to waste time if it wasn't good enough